### PR TITLE
Mapping Checks & Map Fixes

### PR DESCRIPTION
**Pull request by MarkSuckerberg**
## About The Pull Request
Integrates the plumbing and wiring checks into the ship placement unit test. Should prevent and maintainer-destroying spurious CI failures.

Also re-enables atmospherics in unit tests.

## Why It's Good For The Game
Makes sure maps are in proper order before letting them be merged.

## Changelog
:cl:
code: Adds strict mapping checks to unit tests.
/:cl:

### CODE REVIEW

**Comment by MarkSuckerberg** [comment]
how, literally how

is the double piping not the cause of the runtime now????

**Comment by MarkSuckerberg** [comment]
This is the one time I *didn't* want to see green checkmarks. This means the issue is deeper than just doublepipes. FML

**Comment by MarkSuckerberg** [comment]
Whoops

File: code/modules/admin/verbs/atmosdebug.dm [hunk 1]
```diff
@@ -1,59 +1,85 @@
...
+	var/shuttle_dir = turn(shuttle.port_direction, -dir2angle(shuttle.dir)) // evil code
+	switch(shuttle_dir)
+		if(SOUTH)
+			return list(x + 1 - min(bounds[1], bounds[3]), y + 1 - min(bounds[2], bounds[4]))
+		if(NORTH)
+			return list(-(x - 1 - max(bounds[1], bounds[3])), y + 1 - min(bounds[2], bounds[4]))
+		if(EAST)
+			return list(y + 1 - min(bounds[2], bounds[4]), x + 1 - min(bounds[1], bounds[3]))
+		if(WEST)
+			return list(y + 1 - min(bounds[2], bounds[4]), -(x - 1 - max(bounds[1], bounds[3])))
```
Comment (MarkSuckerberg) [inline code/modules/admin/verbs/atmosdebug.dm]: I'm going to be blunt, this *works* but I really need help to make this actually work correctly 100% of the time. If anyone is willing and able to help, please reach out to me, the math is very much off and the macro is just not how it should be.

**Comment by MarkSuckerberg** [comment]
Updated, conflicts resolved, still looking for help with this

**Review by francinum — APPROVED** [review]

**Pull request merged**